src/sas/qtgui/Plotting/SlicerParameters.py

```diff
@@ -216,6 +284,30 @@ def onSlicerChanged(self, index):
                 self.parent.setSlicer(slicer=slicer)
         self.onParamChange()
 
+    def onSlicerSelected(self, item):
+        """
+        Update parameter list when a slicer is selected from lstSlicers
+        """
+        if item.checkState() == QtCore.Qt.Checked:
+            # Uncheck all other items (radio button behavior)
+            for row in range(self.lstSlicers.count()):
+                other_item = self.lstSlicers.item(row)
+                if other_item != item:
+                    other_item.setCheckState(QtCore.Qt.Unchecked)
+
+            # Get the selected slicer name
+            slicer_name = item.text()
+
+            # Update the parameter model to show this slicer's parameters
+            if slicer_name in self.slicer_models:
+                self.setModel(self.slicer_models[slicer_name])
+            elif slicer_name in self.parent.slicers:
+                # Get the model from the slicer object
+                slicer_obj = self.parent.slicers[slicer_name]
+                if hasattr(slicer_obj, '_model'):
+                    self.slicer_models[slicer_name] = slicer_obj._model
```

Comment:
it would probably be safe to also check if `slicer_name` is in `slicer_models`, since you're querying a parent

```python
if hasattr(slicer_obj, '_model') and slicer_name in self.slicer_models:
```